cmd/log_drains.go

```diff
@@ -27,7 +36,19 @@ var (
 				return
 			}
 
-			err := log_drains.List(currentApp)
+			var addonID string
+			if c.GlobalString("addon") != "<addon_id>" {
```

Comment:
There is already a few commands which have a `addon` flag. Please have a look at how they handle it. For instance in the file `cmd/databases.go` on line 40.

Reply:
We cannot use the same method because `addonName` makes the `addon_id` mandatory